recipe/meta.yaml

```diff
@@ -21,7 +21,7 @@ requirements:
     - setuptools
   run:
     - python
-    - numpy
+    - numpy x.x
```

Comment:
Are you sure about this? I don't see `np.get_include()` anywhere to justify the dependency on `numpy`'s ABI. Also, the packages does not even need `numpy` at build time, which shows it does not depended on the `ABI ` :wink: